corva/application.py

```diff
@@ -11,23 +12,31 @@
 class Corva:
     def __init__(
          self,
+         context: Any,
+         *,
          timeout: Optional[int] = None,
          max_retries: Optional[int] = None,
          cache_settings: Optional[dict] = None
     ):
         """
         params:
+         context: AWS Lambda context object
          timeout: api request timeout, set None to use default value
          max_retries: number or api retries for failed request, set to None to use default value
          cache_settings: additional cache settings
         """
 
+        client_context = CorvaLambdaClientContext.from_context(context)
+
+        if (api_key := client_context.api_key or SETTINGS.API_KEY) is None:
```

Comment:
maybe let's avoid new syntax, since we want this library to work with older python versions?

Reply:
fixed